From 77cd176741fd45a78bd8c30539d7f4e6c7f73550 Mon Sep 17 00:00:00 2001
From: Danh Duong <danhdc1@fsoft.com.vn>
Date: Wed, 23 Jan 2019 17:13:00 +0700
Subject: mmngr: Fix invalid type for argument variable of function

Due wrong type of argument variable in function.
This leads the compiler can not execute successfully

Signed-off-by: Danh Duong <danhdc1@fsoft.com.vn>
---
 drv/mmngr_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drv/mmngr_drv.c b/drv/mmngr_drv.c
index ba136f7..9a26dcd 100755
--- a/drv/mmngr_drv.c
+++ b/drv/mmngr_drv.c
@@ -847,7 +847,7 @@ static long ioctl(struct file *file, unsigned int cmd, unsigned long arg)
		}
		break;
	case MM_IOC_VTOP:   /* change virtual address to physical address */
-		mm_cnv_addr((int __user *)arg);
+		mm_cnv_addr((struct MM_PARAM *)arg);
		break;
	default:
		printk(KERN_ERR "MMD CMD EFAULT\n");
--
2.7.4
